web/app/dashboard/dashboard.component.scss

```diff
@@ -19,3 +19,16 @@
         }
     }
 }
+
+.fci-project-table {
+    .mat-header-row {
+        font-family: 'Google Sans', sans-serif;
```

Comment:
Is the long term plan to ship our own font?

Reply:
I have a feeling we'll end up using Roboto. I'm just sticking to the pre-loaded fonts for now.

Reply:
Perfect 👍